python/rascaline/utils.py

```diff
@@ -20,3 +22,225 @@ def _call_with_growing_buffer(callback, initial=1024):
             else:
                 raise
     return buffer.value.decode("utf8")
+
+
+def convert_old_hyperparameter_names(hyperparameters, mode="librascal"):
```

Comment:
I would not put a default here, instead either guess from the hypers or force the user to specify what they want

```suggestion
def convert_old_hyperparameter_names(hyperparameters, mode):
```